src/pages/VillainShow.js

```diff
@@ -1,12 +1,13 @@
 import React from "react"
-import { useParams } from "react-router-dom"
-import {Card, CardBody, CardSubtitle, CardText, CardTitle} from 'reactstrap'
+import {Card, CardBody, CardSubtitle, CardText, CardTitle, Button} from 'reactstrap'
+import { NavLink, useParams } from "react-router-dom"
+// import {currentVillain} from "./src/pages/VillainEdit"
 
 const VillainShow = ({ villains }) => {
   const { id } = useParams()
   let selectedVillain = villains.find((villain) => villain.id === +id)
+  let currentVillain = villains?.find((villain) => villain.id === +id)
```

Comment:
This looks the same as line 8, do you need both of these?

Reply:
deleted